### DISCUSSION TITLE
Access GPTs?

### DISCUSSION THREAD

**Discussion by abdessalaam** [danny-avila/LibreChat#2957] [Q&A] [answered]
Is there a way to access GPTs, that would currently work with ChatGPT plus, with LibreChat? As an example, I am looking to try this:
[Arabic Mentor on ChatGPT Plus](https://chat.openai.com/g/g-6iVaMcXsU-arabic-mentor)
[Arabic Mentor Description](https://gptstore.ai/gpts/K9vB9g9wEA-arabic-mentor)

**Comment by danny-avila** [comment, marked as answer]
No, you would have to make your own versions of them with Assistants
